src/auth/auth.js

```diff
@@ -1,4 +1,6 @@
+import axios from 'axios'
 import {parseJwt, getRandomString} from './jwt.js'
+import Vue from 'vue'
 
```

Comment:
related to auth.js, could you check if that getId function needs to be fixed, as that is missing return.

Reply:
Good catch. It was broken in multiple ways and not used anywhere so I removed it now.